R/fct_SQLite.R

```diff
@@ -369,3 +496,72 @@ db_get_review <- function(
       dplyr::as_tibble()
   })
 }
+
+update_db_version <- function(db_path, version = "1.1") {
+  stopifnot(file.exists(db_path))
+  version <- match.arg(version)
+  temp_path <- withr::local_tempfile(fileext = ".sqlite")
+  file.copy(db_path, temp_path)
+  con <- get_db_connection(temp_path)
+  
+  current_version <- tryCatch({
+    DBI::dbGetQuery(con, "SELECT version FROM db_version") |> 
+      unlist(use.names = FALSE)}, error = \(e){""})
```

Comment:
Maybe it would be helpful to extract this code as a small helper function db_get_version(), so that it is easy to check the db version.

Reply:
I don't really think adding a function would simplify anything. I just copied how you handled this in `db_update()`.